dists/go-ipfs/mkdist.js

```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env node
```

Comment:
yeah, this broke on two of my machines because my binary (from my package manager) was called `nodejs`